NOTICE

```diff
@@ -1 +1,6 @@
 Copyright (c) 2017-2019 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+This software contains source code from Apache BookKeeper, distributed under
```

Comment:
Please do not touch the NOTICE file, this is for legal compliance and we shouldn't be changing it as part of this PR.

Reply:
@fpj My understanding was that we needed this statement in the `NOTICE` file to address copyright concerns. No?

> ...
> Add a note in the NOTICE file saying that this software includes software copyrighted to the ASF from Apache BookKeeper.
> ...

Reply:
I didn't read the addition carefully, you are right.